client/src/components/InputSelect/InputSelect.css

```diff
@@ -0,0 +1,18 @@
+.InputSelect {
+  width: 20rem;
+  height: 3rem;
+  background: #fafafa;
+  border: 2px solid #e5e5e5;
```

Comment:
For colors, if you update your branch with `develop`, you should get the CSS variables that Bill created. Do you mind using those for the color and font properties?